src/pal/test/snp_attestation_validation.cpp

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+
+#include "ccf/ds/hex.h"
+#include "ccf/ds/quote_info.h"
+#include "ccf/pal/attestation.h"
+#include "ccf/pal/measurement.h"
+#include "ccf/pal/report_data.h"
+#include "crypto/openssl/hash.h"
+#include "pal/test/snp_attestation_validation_data.h"
+
+#define DOCTEST_CONFIG_IMPLEMENT
+#include <doctest/doctest.h>
+
+TEST_CASE("milan validation")
+{
+  using namespace ccf;
+
+  auto milan_quote_info = QuoteInfo{
+    .format = QuoteFormat::amd_sev_snp_v1,
+    .quote = pal::snp::testing::milan_attestation,
+    // bundle of certificates
+    // chip_certificate /o sev_version_certificate /o root_certificate
+    // root pubkey -> root cert -> sev_version_cert (ASK?) -> chip_cert
+    // sig algo of attestation sig must be ecdsa_p384_sha384
```

Comment:
Is that something that's derivable from the cert? If not, can we have a pointer to the bit of the spec that hardcodes it?